cookbooks/scale_apache/recipes/default.rb

```diff
@@ -234,16 +234,30 @@
   node.default['fb_apache']['sites']['_default_:443'][key] = val
 end
 
-pkgs = %w{
-  git
-  php
-  php-gd
-  php-mysql
-  php-pdo
-  php-xml
-  php-mbstring
-  python2-boto
-}
+if node.centos7?
+  pkgs = %w{
+    git
+    php
+    php-gd
+    php-mysql
+    php-pdo
+    php-xml
+    php-mbstring
+    python2-boto
+  }
+elsif node.centos8?
+  pkgs = %w{
+    git
+    php
+    php-gd
+    php-json
+    php-mysqlnd
+    php-pdo
+    php-xml
+    php-mbstring
+    python3-boto3
+  } 
+end
```

Comment:
Consolidate copypasta and future proof
```suggestion
pkgs = %w{
  git
  php
  php-gd
  php-mysql
  php-pdo
  php-xml
  php-mbstring
}
if node.centos7?
  pkgs << 'python2-boto'
else
  pkgs << 'python3-boto3'
end
```